### PR TITLE
CLDR-14716 keyboard editorial

### PR DESCRIPTION
**Pull request by miloush**
CLDR-14716

- [X] This PR completes the ticket.

### CODE REVIEW

**Comment by srl295** [comment]
fyi @unicode-org/cldr-keyboard

File: docs/ldml/tr35-keyboards.md [hunk 1]
```diff
@@ -977,6 +988,15 @@ _Attribute:_ `type` (required)
 
 > Current values: `simple`, `final`.
 
+
+There are other keying behaviors that are needed particularly in handling languages and scripts from various parts of the world. The behaviors intended to be covered by the transforms are:
```
Comment (xadxura) [inline docs/ldml/tr35-keyboards.md:992]: "handling languages and scripts from various parts of the worlds" seems a bit vague. How about shortening to "handing complex orthographies".

**Review by xadxura — APPROVED** [review]

File: docs/ldml/tr35-keyboards.md (hunk 1 above)
Comment (miloush) [reply docs/ldml/tr35-keyboards.md]: updated

**Comment by srl295** [comment]
Needs review then we can merge 🎉

**Comment by srl295** [comment]
👉  Review Note:  https://github.com/miloush/cldr/compare/b55932d..a464233#diff-8ae27b7df7b748f4d1f3fd13aba1a28451d42afd704d38a275bddebe82089aa3  seems to produce a diff across the force push for the specific .md file. (rich diff not available)

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**